### PR TITLE
loading

### PR DESCRIPTION
**Pull request by Yordinia**
Set up to do list based on the webpack.
Create an empty To Do List placeholder div or ul element .
Create an index.js file and set an array of some simple to do tasks (array of objects);
Each task object should contain three keys:
- description [string]
- completed [bool]
- index: [number]
Write a function to iterate over the tasks array and populate an HTML list item element for each task.
render the dynamically created list of tasks in the dedicated placeholder.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import './style.css';
+
+let todoTask = [];
```
Comment (BenMukebo) [inline src/index.js:3]: - [ ] According to the project requirement, should have created an array of some simple *to do tasks* (array of objects) with each object containing three keys - description, completed, and index. 👇🏻 

im2


- Kindly make sure you address this. 💯 👍🏻  Your code might look like this 👍🏻

```javascript
const tasks = [
  {
    description: '...',
    completed: false,
    index: 10,
  },
];
```

- Note:  On page load, the task array is rendered dynamically to the DOM. and you should make sure The tasks should appear in order of the `index` values for each task. 😉 

### NOTE: Adding and removing TODOs is not part of the project yet!

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Yordinia,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Webpack dev server is set up and running with no errors.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

00Mit

 The reason for this is that there is no `MIT.md` file in the root directory of this project.  Please, and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/index.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Yordinia 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**